index.js

```diff
@@ -14,6 +14,7 @@
     .provider('Analytics', function () {
       var accounts,
           analyticsJS = true,
+          appName,
```

Comment:
Also, please expose this value in the configuration property of the return object. All variables are exposed there (near the end of the file). This variable also needs to be listed in the README.md as available under the configuration property.